src/test/java/org/apache/commons/text/AlphabetConverterTest.java

```diff
@@ -201,4 +207,97 @@ private void test(final Character[] originalChars, final Character[] encodingCha
             Assert.assertEquals("Encoded '" + s + "' into '" + encoded + "', but decoded into '" + decoded + "'", s, decoded);
         }
     }
+
+    @Test
+    public void testCreateConverterFromCharsWithNullAndNull() {
+
```

Comment:
Blank lines at the start of methods, like this one, seem inconsistent with the rest of the code base.

Please consider removing them from _all_ methods which you've created and touched.